src/utils/eventUtils.ts

```diff
@@ -0,0 +1,16 @@
+import type { CellEvent } from '../types';
+
+export function createCellEvent<E extends React.SyntheticEvent<HTMLDivElement>>(
+  event: E
+): CellEvent<E> {
+  let defaultPrevented = false;
+  return {
+    ...event,
```

Comment:
We cannot call `preventDefault()` when we need to skip tab key. Calling `preventDefault` will prevent the default behavior and next element won't be focused. Cannot think of another way other than extending the default event.